client/stylesheets/_toCat.scss

```diff
@@ -5604,12 +5608,14 @@ On click the images will be displayed at normal size to complete the effect
     padding: 10px 15px;
     color: #fff !important;
     border-radius: 4px;
-    background-color: #404040 !important;
+    background-color: #f8504b !important;
     margin-left: 0;
 }
 
 .profileUserAmount ul li span {
     display: inline-block !important;
+    font-family: 'Helvetica Neue', sans-serif !important;
```

Comment:
!important should not be used<br>Line should be indented 2 spaces, but was indented 4 spaces